var/spack/repos/builtin/packages/pacparser/package.py

```diff
@@ -9,9 +9,13 @@
 class Pacparser(MakefilePackage):
     """pacparser is a library to parse proxy auto-config (PAC) files."""
 
+    maintainers = ['iarspider']
+
     homepage = "https://pacparser.github.io/"
     url      = "https://github.com/manugarg/pacparser/releases/download/1.3.7/pacparser-1.3.7.tar.gz"
 
+    version('1.3.8', sha256='4e2872de565b2b64ffc81ba503e0eba35b3f7ef4a023ddd4a328c7b9d2cac266',
+            url='https://github.com/manugarg/pacparser/releases/download/v1.3.8/pacparser-v1.3.8.tar.gz')
```

Comment:
Can we update the `url` and use a specific `url` in v1.3.7 ? Asking under the assumption that the new naming scheme is what will be used in the next versions.

Reply:
Looks like new naming scheme will indeed be `v<version>`. Changed, added 1.3.9 (no URL, only a tag)